### PR TITLE
names : add firstname, lastname and affiliation to sign up + remove username of other users from display

### PR DESCRIPTION
**Pull request by SandraMonnier**

### CODE REVIEW

File: colab-api/src/main/java/ch/colabproject/colab/api/controller/user/UserManager.java [hunk 1]
```diff
@@ -201,78 +200,80 @@ public AuthMethod getAuthenticationMethod(String identifier) {
...
      * @param plainPassword plain text password
      *
-     * @return a brand new user
+     * @return a brand-new user
      *
      * @throws HttpErrorMessage if username is already taken
      */
     public User createAdminUser(String username, String email, String plainPassword) {
-        User admin = this.createUser(username, email, plainPassword);
+        SignUpInfo signUpInfo = new SignUpInfo();
```
Comment (TehAwol) [inline colab-api/src/main/java/ch/colabproject/colab/api/controller/user/UserManager.java:261]: Why not pass signUpInfo directly like in createUser() ?

**Review by TehAwol — COMMENTED** [review]
UserManager.java : comment for createUser vs createAdminUser

File: colab-api/src/main/java/ch/colabproject/colab/api/controller/user/UserManager.java (hunk 1 above)
Comment (SandraMonnier) [reply colab-api/src/main/java/ch/colabproject/colab/api/controller/user/UserManager.java]: Finally, I changed the structure of the method to be smarter

**Review by SandraMonnier — COMMENTED** [review]
self check OK

**Pull request merged**